packages/manager/apps/restricted/src/hooks/useMinimalRights/constants.ts

```diff
@@ -0,0 +1,14 @@
+type RIGHT = {
+  urn: string;
+  subsidiariesExcluded?: string[];
```

Comment:
If we want the perfection 🤓 : 

```suggestion
  subsidiariesExcluded?: ('EU' | 'CA' | 'US')[];
```

But we have not good generic type for subsidiaries  😢

Reply:
there are regions here but as it's not provided by the app so I have to use subsidiary instead (the value can be different of those 3 then)